tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+
+@pytest.mark.integration_test
+def test_integration_wave_01_02_03():
+    # make a vendor  
+    vendor = Vendor()
+    assert len(vendor.inventory) == 0
+
+    # add an item
+    item1 = Item(category="Clothing")
+    item2 = Item(category="Electronics")
+    result1 = vendor.add(item1)
+    result2 = vendor.add(item2)
+
+    assert len(vendor.inventory) == 2
+    assert item1 in vendor.inventory
+    assert item2 in vendor.inventory
+    assert result1 == item1
+    assert result2 == item2
+
+    # remove an item
+    result = vendor.remove(item1)
```

Comment:
Since we have `result1` and `result2` above, does it make sense for this to be something like `result3` or `remove_result`?